python/doc/examples/reliability_sensitivity/sensitivity_sobol.ipynb

```diff
@@ -29,29 +29,29 @@
    "source": [
     "## Introduction\n",
     "\n",
-    "In this example we are going to quantify the correlation between the input variables and the output variable of a model thanks to the Sobol indices.\n",
+    "In this example we are going to quantify the correlation between the input variables and the output variable of a model thanks to Sobol indices.\n",
     "\n",
-    "The Sobol indices allow to evaluate the importance of a single variable or a specific set of variables. Here the Sobol indices are estimated by sampling, from two input samples and a numerical function.\n",
+    "Sobol indices are designed to evaluate the importance of a single variable or a specific set of variables. Here the Sobol indices are estimated by sampling from the two input samples and propagating uncertainty through a numerical function.\n",
     "\n",
-    "In theory, Sobol indices range is $\\left[0; 1\\right]$ ; the more the indice value is close to 1 the more the variable is important toward the output of the function. \n",
+    "In theory, Sobol indices range from 0 to 1; the closer an index value is to 1, the better the associated input variable explains the function output. \n",
     "\n",
     "Let us denote by $d$ the input dimension of the model. \n",
     "\n",
     "The Sobol' indices can be computed at different orders.\n",
     "\n",
-    "* The first order indices evaluate the importance of one variable at a time.\n",
+    "* The first order indices evaluate the importance of one input variable at a time.\n",
     "\n",
-    "* The $d$ total indices give the relative importance of every variables except the variable $X_i$, for every variable.\n",
+    "* The total indices give the relative importance of one input variable and all its interactions with other variables. Alternatively, they can be viewed as measuring how much wriggle room remains to the output when all but one input variables are fixed.\n",
     "\n",
-    "* In general, we are only interested in first order and total Sobol' indices. There are situations, however, where we want to estimate interactions. The second order indices evaluate the importance of every pair of variables; this represents \n",
+    "* In general, we are only interested in first order and total Sobol' indices. There are situations, however, where we want to estimate interactions. The second order indices evaluate the importance of every pair of input variables variables; this represents \n",
```

Comment:
```suggestion
    "* In general, we are only interested in first order and total Sobol' indices. There are situations, however, where we want to estimate interactions. Second order indices evaluate the importance of every pair of input variables; there are \n",
```
I made a mistake when doing the original changes to this file: I wrote one too many "variables" in the sentence.

Reply:
Fixed.